vfx/gl/texture_program.cc

```diff
@@ -0,0 +1,60 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "vfx/gl/texture_program.h"
+
+#include <memory>
+
+#include "ui/gl/gl_bindings.h"
+
+namespace vfx {
+namespace {
+
+const char kVertexShaderSource[] = R"GLSL(
+uniform mat4 u_transform;
+attribute vec4 a_position;
+attribute vec4 a_color;
+attribute vec2 a_tex_coord;
+
+varying vec2 v_tex_coord;
+varying vec4 v_color;
+
+void main() {
+  gl_Position = u_transform * a_position, 1.0;
```

Comment:
Oops.  This is typo.  It's just supposed to be

`gl_Position = u_transform * a_position`